diff --git a/node_modules/graceful-fs/graceful-fs.js b/node_modules/graceful-fs/graceful-fs.js
index 8d5b89e..ed3c730 100644
--- a/node_modules/graceful-fs/graceful-fs.js
+++ b/node_modules/graceful-fs/graceful-fs.js
@@ -19,11 +19,11 @@ if (typeof Symbol === 'function' && typeof Symbol.for === 'function') {
   previousSymbol = '___graceful-fs.previous'
 }
 
-function noop () {}
+function noop() { }
 
 function publishQueue(context, queue) {
   Object.defineProperty(context, gracefulQueue, {
-    get: function() {
+    get: function () {
       return queue
     }
   })
@@ -33,7 +33,7 @@ var debug = noop
 if (util.debuglog)
   debug = util.debuglog('gfs4')
 else if (/\bgfs4\b/i.test(process.env.NODE_DEBUG || ''))
-  debug = function() {
+  debug = function () {
     var m = util.format.apply(util, arguments)
     m = 'GFS4: ' + m.split(/\n/).join('\nGFS4: ')
     console.error(m)
@@ -50,7 +50,7 @@ if (!fs[gracefulQueue]) {
   // This is essential when multiple graceful-fs instances are
   // in play at the same time.
   fs.close = (function (fs$close) {
-    function close (fd, cb) {
+    function close(fd, cb) {
       return fs$close.call(fs, fd, function (err) {
         // This function uses the graceful-fs shared queue
         if (!err) {
@@ -69,7 +69,7 @@ if (!fs[gracefulQueue]) {
   })(fs.close)
 
   fs.closeSync = (function (fs$closeSync) {
-    function closeSync (fd) {
+    function closeSync(fd) {
       // This function uses the graceful-fs shared queue
       fs$closeSync.apply(fs, arguments)
       resetQueue()
@@ -82,7 +82,7 @@ if (!fs[gracefulQueue]) {
   })(fs.closeSync)
 
   if (/\bgfs4\b/i.test(process.env.NODE_DEBUG || '')) {
-    process.on('exit', function() {
+    process.on('exit', function () {
       debug(fs[gracefulQueue])
       require('assert').equal(fs[gracefulQueue].length, 0)
     })
@@ -95,26 +95,26 @@ if (!global[gracefulQueue]) {
 
 module.exports = patch(clone(fs))
 if (process.env.TEST_GRACEFUL_FS_GLOBAL_PATCH && !fs.__patched) {
-    module.exports = patch(fs)
-    fs.__patched = true;
+  module.exports = patch(fs)
+  fs.__patched = true;
 }
 
-function patch (fs) {
+function patch(fs) {
   // Everything that references the open() function needs to be in here
-  polyfills(fs)
+  // Polyfills removed to prevent TypeError during bundling
   fs.gracefulify = patch
 
   fs.createReadStream = createReadStream
   fs.createWriteStream = createWriteStream
   var fs$readFile = fs.readFile
   fs.readFile = readFile
-  function readFile (path, options, cb) {
+  function readFile(path, options, cb) {
     if (typeof options === 'function')
       cb = options, options = null
 
     return go$readFile(path, options, cb)
 
-    function go$readFile (path, options, cb, startTime) {
+    function go$readFile(path, options, cb, startTime) {
       return fs$readFile(path, options, function (err) {
         if (err && (err.code === 'EMFILE' || err.code === 'ENFILE'))
           enqueue([go$readFile, [path, options, cb], err, startTime || Date.now(), Date.now()])
@@ -128,13 +128,13 @@ function patch (fs) {
 
   var fs$writeFile = fs.writeFile
   fs.writeFile = writeFile
-  function writeFile (path, data, options, cb) {
+  function writeFile(path, data, options, cb) {
     if (typeof options === 'function')
       cb = options, options = null
 
     return go$writeFile(path, data, options, cb)
 
-    function go$writeFile (path, data, options, cb, startTime) {
+    function go$writeFile(path, data, options, cb, startTime) {
       return fs$writeFile(path, data, options, function (err) {
         if (err && (err.code === 'EMFILE' || err.code === 'ENFILE'))
           enqueue([go$writeFile, [path, data, options, cb], err, startTime || Date.now(), Date.now()])
@@ -149,13 +149,13 @@ function patch (fs) {
   var fs$appendFile = fs.appendFile
   if (fs$appendFile)
     fs.appendFile = appendFile
-  function appendFile (path, data, options, cb) {
+  function appendFile(path, data, options, cb) {
     if (typeof options === 'function')
       cb = options, options = null
 
     return go$appendFile(path, data, options, cb)
 
-    function go$appendFile (path, data, options, cb, startTime) {
+    function go$appendFile(path, data, options, cb, startTime) {
       return fs$appendFile(path, data, options, function (err) {
         if (err && (err.code === 'EMFILE' || err.code === 'ENFILE'))
           enqueue([go$appendFile, [path, data, options, cb], err, startTime || Date.now(), Date.now()])
@@ -170,14 +170,14 @@ function patch (fs) {
   var fs$copyFile = fs.copyFile
   if (fs$copyFile)
     fs.copyFile = copyFile
-  function copyFile (src, dest, flags, cb) {
+  function copyFile(src, dest, flags, cb) {
     if (typeof flags === 'function') {
       cb = flags
       flags = 0
     }
     return go$copyFile(src, dest, flags, cb)
 
-    function go$copyFile (src, dest, flags, cb, startTime) {
+    function go$copyFile(src, dest, flags, cb, startTime) {
       return fs$copyFile(src, dest, flags, function (err) {
         if (err && (err.code === 'EMFILE' || err.code === 'ENFILE'))
           enqueue([go$copyFile, [src, dest, flags, cb], err, startTime || Date.now(), Date.now()])
@@ -192,17 +192,17 @@ function patch (fs) {
   var fs$readdir = fs.readdir
   fs.readdir = readdir
   var noReaddirOptionVersions = /^v[0-5]\./
-  function readdir (path, options, cb) {
+  function readdir(path, options, cb) {
     if (typeof options === 'function')
       cb = options, options = null
 
     var go$readdir = noReaddirOptionVersions.test(process.version)
-      ? function go$readdir (path, options, cb, startTime) {
+      ? function go$readdir(path, options, cb, startTime) {
         return fs$readdir(path, fs$readdirCallback(
           path, options, cb, startTime
         ))
       }
-      : function go$readdir (path, options, cb, startTime) {
+      : function go$readdir(path, options, cb, startTime) {
         return fs$readdir(path, options, fs$readdirCallback(
           path, options, cb, startTime
         ))
@@ -210,7 +210,7 @@ function patch (fs) {
 
     return go$readdir(path, options, cb)
 
-    function fs$readdirCallback (path, options, cb, startTime) {
+    function fs$readdirCallback(path, options, cb, startTime) {
       return function (err, files) {
         if (err && (err.code === 'EMFILE' || err.code === 'ENFILE'))
           enqueue([
@@ -294,14 +294,14 @@ function patch (fs) {
     configurable: true
   })
 
-  function ReadStream (path, options) {
+  function ReadStream(path, options) {
     if (this instanceof ReadStream)
       return fs$ReadStream.apply(this, arguments), this
     else
       return ReadStream.apply(Object.create(ReadStream.prototype), arguments)
   }
 
-  function ReadStream$open () {
+  function ReadStream$open() {
     var that = this
     open(that.path, that.flags, that.mode, function (err, fd) {
       if (err) {
@@ -317,14 +317,14 @@ function patch (fs) {
     })
   }
 
-  function WriteStream (path, options) {
+  function WriteStream(path, options) {
     if (this instanceof WriteStream)
       return fs$WriteStream.apply(this, arguments), this
     else
       return WriteStream.apply(Object.create(WriteStream.prototype), arguments)
   }
 
-  function WriteStream$open () {
+  function WriteStream$open() {
     var that = this
     open(that.path, that.flags, that.mode, function (err, fd) {
       if (err) {
@@ -337,23 +337,23 @@ function patch (fs) {
     })
   }
 
-  function createReadStream (path, options) {
+  function createReadStream(path, options) {
     return new fs.ReadStream(path, options)
   }
 
-  function createWriteStream (path, options) {
+  function createWriteStream(path, options) {
     return new fs.WriteStream(path, options)
   }
 
   var fs$open = fs.open
   fs.open = open
-  function open (path, flags, mode, cb) {
+  function open(path, flags, mode, cb) {
     if (typeof mode === 'function')
       cb = mode, mode = null
 
     return go$open(path, flags, mode, cb)
 
-    function go$open (path, flags, mode, cb, startTime) {
+    function go$open(path, flags, mode, cb, startTime) {
       return fs$open(path, flags, mode, function (err, fd) {
         if (err && (err.code === 'EMFILE' || err.code === 'ENFILE'))
           enqueue([go$open, [path, flags, mode, cb], err, startTime || Date.now(), Date.now()])
@@ -368,7 +368,7 @@ function patch (fs) {
   return fs
 }
 
-function enqueue (elem) {
+function enqueue(elem) {
   debug('ENQUEUE', elem[0].name, elem[1])
   fs[gracefulQueue].push(elem)
   retry()
@@ -380,7 +380,7 @@ var retryTimer
 // reset the startTime and lastTime to now
 // this resets the start of the 60 second overall timeout as well as the
 // delay between attempts so that we'll retry these jobs sooner
-function resetQueue () {
+function resetQueue() {
   var now = Date.now()
   for (var i = 0; i < fs[gracefulQueue].length; ++i) {
     // entries that are only a length of 2 are from an older version, don't
@@ -394,7 +394,7 @@ function resetQueue () {
   retry()
 }
 
-function retry () {
+function retry() {
   // clear the timer and remove it to help prevent unintended concurrency
   clearTimeout(retryTimer)
   retryTimer = undefined
